playbooks/common/openshift-master/config.yml

```diff
@@ -232,6 +232,7 @@
       balance: source
       servers: "{{ hostvars.localhost.haproxy_backend_servers }}"
   roles:
+  - role: openshift_facts
```

Comment:
These hosts aren't gathering facts prior to this play (now that `oo_lb_to_config` is absent from docker hosts) so `is_containerized` won't be set for `haproxy` and `os_firewall` roles.